data/migrations/v1.3.0-polygon.sql

```diff
@@ -3,11 +3,48 @@ UPDATE
   SET mz_poi_min_zoom = mz_calculate_min_zoom_pois(p.*),
       mz_landuse_min_zoom = mz_calculate_min_zoom_landuse(p.*)
   WHERE
-    (tags -> 'leisure' = 'garden' OR
-     tags -> 'landuse' = 'village_green' OR
-     tags -> 'natural' IN ('wood', 'forest') OR
-     tags -> 'man_made' = 'pier' OR
-     (NOT tags ? 'name' AND mz_poi_min_zoom IS NOT NULL))
+    (
+      tags -> 'aerialway' = 'pylon'
+      OR tags -> 'aeroway' IN ('gate', 'helipad')
+      OR tags -> 'amenity' IN ('atm', 'bbq', 'bench', 'bicycle_parking', 'bicycle_rental',
+        'bicycle_repair_station', 'boat_storage', 'car_sharing', 'fuel',
+        'life_ring', 'parking', 'picnic_table', 'post_box', 'ranger_station', 'recycling',
+        'shelter', 'shower', 'telephone', 'toilets', 'waste_basket', 'waste_disposal',
+        'water_point', 'watering_place')
+      OR tags -> 'barrier' IN ('cycle_barrier', 'gate', 'toll_booth')
+      OR tags -> 'emergency' IN ('lifeguard_tower', 'phone')
+      OR tags -> 'highway' IN ('bus_stop', 'ford', 'mini_roundabout', 'motorway_junction',
+        'platform', 'rest_area', 'traffic_signals', 'trailhead')
+      OR tags -> 'historic' = 'landmark'
+      OR tags -> 'landuse' IN ('village_green', 'quarry')
+      OR tags -> 'leisure' IN ('garden', 'dog_park', 'firepit', 'fishing', 'pitch',
+        'slipway', 'swimming_area')
+      OR tags -> 'lock' = 'yes'
+      OR tags -> 'man_made' IN ('adit', 'communications_tower', 'mast', 'mineshaft',
+        'observatory', 'offshore_platform', 'petroleum_well', 'power_wind', 'telescope',
+        'water_tower', 'water_well', 'windmill')
+      OR tags -> 'natural' IN ('cave_entrance', 'peak', 'volcano', 'forest', 'geyser',
+        'hot_spring', 'rock', 'saddle', 'stone', 'spring', 'tree', 'waterfall', 'wood')
+      OR tags -> 'power' IN ('pole', 'tower')
+      OR tags -> 'public_transport' IN ('platform', 'stop_area')
+      OR tags -> 'railway' IN ('halt', 'level_crossing', 'platform', 'stop',
+        'subway_entrance', 'tram_stop')
+      OR tags -> 'tags->icn_ref' = 'true'
+      OR tags -> 'tags->iwn_ref' = 'true'
+      OR tags -> 'tags->lcn_ref' = 'true'
+      OR tags -> 'tags->lwn_ref' = 'true'
+      OR tags -> 'tags->ncn_ref' = 'true'
+      OR tags -> 'tags->nwn_ref' = 'true'
+      OR tags -> 'tags->rcn_ref' = 'true'
+      OR tags -> 'tags->rwn_ref' = 'true'
+      OR tags -> 'tags->whitewater' IN ('egress', 'hazard', 'put_in', 'put_in;egress',
+        'rapid')
+      OR tags -> 'tourism' IN ('alpine_hut', 'information', 'picnic_site', 'viewpoint',
+        'wilderness_hut')
+      OR tags -> 'waterway' IN ('dam', 'lock', 'waterfall')
+      OR (NOT tags ? 'name' AND mz_poi_min_zoom IS NOT NULL)
+      OR tags -> 'man_made' = 'pier'
```

Comment:
I think this technically works though it could get merged with general IN `man_made` above. This is probably more legible as `pier` is special.